src/cmake/ov_parallel.cmake

```diff
@@ -398,4 +398,8 @@ function(set_ie_threading_interface_for TARGET_NAME)
         find_package(Threads REQUIRED)
         ie_target_link_libraries(${TARGET_NAME} ${LINK_TYPE} Threads::Threads)
     endif()
+endfunction(set_ov_threading_interface_for)
+
+function(set_ie_threading_interface_for TARGET_NAME)
```

Comment:
let's insert a warning that new macro / function should be used

Reply:
Added